keyboards/handwired/jotpad16/jotpad16.c

```diff
@@ -4,3 +4,10 @@ void matrix_init_kb(void) {
 
 	matrix_init_user();
 }
+
+void keyboard_pre_init_kb() {
+    setPinOutput(JOTPAD16_LED1);
+    setPinOutput(JOTPAD16_LED2);
```

Comment:
You may want to write to the pin too?

Reply:
I want to use the pins only to show the layers/caps lock status with leds.

i added the lines because of this in the documentation:
https://github.com/qmk/qmk_firmware/blob/master/docs/custom_quantum_functions.md#example-keyboard_pre_init_user-implementation